hs_core/models.py

```diff
@@ -718,26 +729,25 @@ def update(cls, element_id, **kwargs):
         metadata_obj = kwargs['content_object']
         # get matching resource
         resource = BaseResource.objects.filter(object_id=metadata_obj.id).first()
-
+        publisher_CUAHSI = "Consortium of Universities for the Advancement of Hydrologic Science, Inc. (CUAHSI)"
```

Comment:
@hyi It looks like this issue will be further discussed in our weekly call this week. Will make necessary changes after that.